mak/COPY

```diff
@@ -93,6 +108,7 @@ COPY=\
 	$(IMPDIR)\core\sys\osx\mach\thread_act.d \
 	\
 	$(IMPDIR)\core\sys\osx\sys\cdefs.d \
+	$(IMPDIR)\core\sys\osx\sys\event.d \
```

Comment:
Confusing to whom?  Because it was never added to `mak/COPY`, it never made it into the `import/` directory in an official release.  The only way someone would use it is if they added `druntime/src` to their path for some reason.

I'm leaning towards just deprecating the `core.sys.osx` modules over time, starting now by sticking doc comments in there indicating they will be deprecated and the `core.sys.darwin` equivalent should be used instead.